.github/workflows/static_analysis.yml

```diff
@@ -43,17 +43,17 @@ jobs:
           java-version: '17'
           distribution: 'microsoft'
 
-      # Assemble artifacts from main branch
-      - name: Checkout Main Branch
+      # Assemble artifacts from Release branch
+      - name: Checkout Release Branch
         uses: actions/checkout@v4
         with:
-          ref: main
-      - name: Assemble Release AAR on Main Branch
+          ref: 1.x
```

Comment:
This GitHub Action compares SDK size with previous versions. I had to update these to create a separate set of artifacts for `1.x` releases because `:PayPalNativePayments` is no longer being built on `main` (this causes the action to fail).

Reply:
Steven, do you need to change it back when you do release for 2.0.0 beta versions?

Reply:
Just to clarify this is only for `1.x` releases and should not affect `main` in any way.